ops/op2/dispatch_fast.rs

```diff
@@ -0,0 +1,195 @@
+  fn quote_type(&self) -> TokenStream {
+    match &self {
+      FastValue::Void => quote!(Type::Void),
+      FastValue::Bool => quote!(Type::Bool),
+      FastValue::U32 => quote!(Type::Uint32),
+      FastValue::I32 => quote!(Type::Int32),
+      FastValue::U64 => quote!(Type::Uint64),
+      FastValue::I64 => quote!(Type::Int64),
+      FastValue::F32 => quote!(Type::Float32),
+      FastValue::F64 => quote!(Type::Float64),
+      FastValue::Pointer => quote!(Type::Pointer),
+      FastValue::V8Value => quote!(Type::V8Value),
+      FastValue::Uint8Array => quote!(Type::TypedArray(CType::Uint8)),
+      FastValue::Uint32Array => quote!(Type::TypedArray(CType::Uint32)),
+      FastValue::Float64Array => quote!(Type::TypedArray(CType::Float64)),
+      FastValue::SeqOneByteString => quote!(Type::SeqOneByteString),
+    }
+  }
+}
+
+pub fn generate_dispatch_fast(
+  generator_state: &mut GeneratorState,
+  signature: &ParsedSignature,
+) -> Result<Option<(TokenStream, TokenStream)>, V8MappingError> {
+  let mut inputs = vec![];
+  for arg in &signature.args {
+    let fv = match arg {
+      Arg::OptionNumeric(_) | Arg::SerdeV8(_) => return Ok(None),
+      Arg::Numeric(NumericArg::bool) => FastValue::Bool,
+      Arg::Numeric(NumericArg::u32)
+      | Arg::Numeric(NumericArg::u16)
+      | Arg::Numeric(NumericArg::u8) => FastValue::U32,
+      Arg::Numeric(NumericArg::i32)
+      | Arg::Numeric(NumericArg::i16)
+      | Arg::Numeric(NumericArg::i8)
+      | Arg::Numeric(NumericArg::__SMI__) => FastValue::I32,
+      Arg::Numeric(NumericArg::u64) | Arg::Numeric(NumericArg::usize) => {
+        FastValue::U64
+      }
+      Arg::Numeric(NumericArg::i64) | Arg::Numeric(NumericArg::isize) => {
+        FastValue::I64
+      }
+      // TODO(mmastrac)
+      Arg::Special(_) => return Ok(None),
+      _ => {
+        return Err(V8MappingError::NoMapping("a fast argument", arg.clone()))
+      }
+    };
+    inputs.push(fv);
+  }
+
+  let ret_val = match &signature.ret_val {
+    RetVal::Infallible(arg) => arg,
+    RetVal::Result(arg) => arg,
+  };
+
+  let output = match ret_val {
+    Arg::OptionNumeric(_) | Arg::SerdeV8(_) => return Ok(None),
+    Arg::Void => FastValue::Void,
+    Arg::Numeric(NumericArg::bool) => FastValue::Bool,
+    Arg::Numeric(NumericArg::u32)
+    | Arg::Numeric(NumericArg::u16)
+    | Arg::Numeric(NumericArg::u8) => FastValue::U32,
+    Arg::Numeric(NumericArg::i32)
+    | Arg::Numeric(NumericArg::i16)
+    | Arg::Numeric(NumericArg::i8) => FastValue::I32,
+    Arg::Numeric(NumericArg::u64) | Arg::Numeric(NumericArg::usize) => {
+      FastValue::U64
+    }
+    Arg::Numeric(NumericArg::i64) | Arg::Numeric(NumericArg::isize) => {
+      FastValue::I64
+    }
+    Arg::Special(_) => return Ok(None),
+    _ => {
+      return Err(V8MappingError::NoMapping(
+        "a fast return value",
+        ret_val.clone(),
+      ))
+    }
+  };
+
+  let GeneratorState {
+    fast_function,
+    deno_core,
```

Comment:
This appears to not be used?

Reply:
They are used in the `quote!` macro below (`#deno_core`, etc)